src/Hydrators/SportHydrator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace TheRealMVP\Hydrators;
+use TheRealMVP\Entities\Sport;
+
+class SportHydrator
+{
+    private \PDO $pdoConnection;
+
+    /**
+     * TeamHydrator constructor - saves PDO connection object as local variable
+     *
+     * @param $pdoConnection
+     */
+    public function __construct(\PDO $pdoConnection)
+    {
+        $this->pdoConnection = $pdoConnection;
+    }
+
+    /**
+     * create database connection and retrieves data and returns an array of sport objects
+     *
+     * @param \PDO $pdoConnection
+     *
+     * @return array
+     */
+    public static function getData(\PDO $pdoConnection) : array
```

Comment:
Same here, we already have a PDO connection as a property so no need to pass it in again

Reply:
Removed constructor function